signer/src/transaction_signer.rs

```diff
@@ -579,10 +584,17 @@ where
         &mut self,
         msg: &message::WstsMessage,
         bitcoin_chain_tip: &model::BitcoinBlockHash,
+        msg_public_key: PublicKey,
+        chain_tip_report: &MsgChainTipReport,
     ) -> Result<(), Error> {
         tracing::info!("handling message");
+
         match &msg.inner {
             wsts::net::Message::DkgBegin(_) => {
+                if !chain_tip_report.sender_is_coordinator {
+                    return Err(Error::NotChainTipCoordinator);
+                }
```

Comment:
For all of these `chain_tip_report.sender_is_coordinator` checks, I imagine that this will happen from time to time, with the consequence being that we sometimes receive an error log message that we have to ignore. Instead I think we should log a warning and return `Ok(())`.

Reply:
Done